### DISCUSSION TITLE
New Livewire page component is always offline, says You are not connected to the internet when I visit it

### DISCUSSION THREAD

**Discussion by DreadfulCode** [livewire/livewire#5546] [1. Help] [answered]
![image](https://user-images.githubusercontent.com/4735134/216854836-28fabf8f-1c24-4a88-8675-73df95ca4c68.png)

Just freshly installed v2.11.2 on the latest of laravel with a component generated via artisan. 

I have this declaration: 
`Route::get('/tickers', TickerTable::class);`
which is registered in web.php, thus any visit to a page via the url, "/tickers", will render a table.

Every request to this new page says I'm offline. "You are not connected to the internet".

What do you recommend for a starting point for troubleshooting this? There is no useful info via Chrome developer tools, because no requests are being made to fetch assets, and no errors in the server logs. No info to go off of.

How do I get my app to enable Livewire to recognize that it's "online"? Which configs control the rendering of a page component?

**Comment by joshhanley** [comment, marked as answer]
@DreadfulCode first thing to make sure of is that you've added Livewire's scripts and styles to your layout file. Once that's done, you can check in dev tools to make sure both are rendering correctly. You can also check Livewire has been initialised properly by typing `Livewire` in your browser console and you should see the Livewire object in response.

If that's all working correctly, then can you share your code for you component class and view so we can have a look?

Hope this helps!
